### DISCUSSION TITLE
Name of Extension doesn't show in Email

### DISCUSSION THREAD

**Discussion by drdingens** [akeeba/panopticon#759] [General]
Hey 
I am receiving email notifications that look like this:
[EXTENSION_NAME] 1.8 is available on
both in the text and in the  subject of the email
![2024_08_02_11_33_35_Mail](https://github.com/user-attachments/assets/2f967f37-c426-4e70-ae94-c8ef595a2107)

Is this a bug or how can I fix this?
Thanks
Rolf

**Comment by nikosdion** [comment]
Do you see the extension name in the web interface?

Which extension is it?

Is this a Joomla, or a WordPress site?

**Comment by drdingens** [comment]
Oh, it’s all extensions, not just one. And yes, I can see all names in the web interface. 
Cheers Rolf

**Comment by drdingens** [comment]
Hey
Any ideas? It's all Wordpress sites
Thanks
Rolf

**Reply by nikosdion** [reply]
Edit the email template “Wordpress Plugin/Theme Update: Update Found” and replace `[EXTENSION_NAME]` with `[SOFTWARE_NAME]` in the subject and body (both HTML and plain text).
